invenio_accounts/ext.py

```diff
@@ -192,6 +192,15 @@ def init_config(self, app):
             if any([k.startswith(prefix) for prefix in config_apps]):
                 app.config.setdefault(k, getattr(config, k))
 
+    def _enable_session_activity(self, app):
+        """Enable session activity."""
+        user_logged_in.connect(login_listener, app)
+        user_logged_out.connect(logout_listener, app)
+        from .views.settings import blueprint
```

Comment:
Do you think can erroneously import? 
I thought was always relative to `invenio_accounts/ext.py`